src/fcurl.c

```diff
@@ -204,11 +206,8 @@ int fcurl_close(struct fcurl_handle *h)
 
 int fcurl_eof(struct fcurl_handle *h)
 {
-  int ret=0;
-
-  /* add code here */
-
-  return ret;
+  /* Still does not make difference between EOF and errors */
+  return (h->transfer_complete && 0 == h->b.used);
```

Comment:
This adds an `feof()` version

Reply:
Yes, a basic one, still does not make difference eof/error, but is needed in a classic while(eof) { read; write; } loop.